kubernetes/samples/charts/traefik/README.md

```diff
@@ -28,7 +28,7 @@ $ helm upgrade traefik-operator traefik/traefik --values values.yaml
 In this section we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
```

Comment:
In this section, we'll (put a comma after the word section.)

Reply:
done